wiki/People/en.md

```diff
@@ -1,13 +1,12 @@
 # People
 
-The following are user groups, made up of the osu!community members that help maintain _osu!_
-Most of these user groups are given forum name colours, in-game chat colours, and/or profile badges.
+The following are user groups, made up of the osu!community members that help maintain osu! Most of these user groups are given forum name colours, in-game chat colours, and/or profile badges or titles.
```

Comment:
`osu!community` -> `osu! community` on the asc.

`help maintain osu!` -> `help maintain osu!.` ?

Reply:
> help maintain osu! -> help maintain osu!. ?

osu! is enough, I believe the `!` acts like a full stop.

Reply:
it's proper grammar, but I think adding the stop makes it much easier to read, since there are lots of words like "osu! \_\_\_\_" on this wiki